go/vt/schemadiff/names.go

```diff
@@ -42,6 +42,9 @@ func ExtractConstraintOriginalName(tableName string, constraintName string) stri
 	if strings.HasPrefix(constraintName, fmt.Sprintf("%s_fk_", tableName)) {
 		return constraintName[len(tableName)+1:]
 	}
+	if strings.HasPrefix(constraintName, fmt.Sprintf("%s_ibfk_", tableName)) {
+		return constraintName[len(tableName)+1:]
+	}
```

Comment:
Do we need the case above this one still? Since that’s not what the automatic names look like?

Reply:
I'll remove it.

Reply:
Removed.